### DISCUSSION TITLE
Adding Flash Changes

### DISCUSSION THREAD

**Discussion by JayTeeLX** [laszlodaniel/ChryslerScanner#16] [Ideas]
Hi Laszlo, 

first of all I want to thank you for your excellent work - I bought a Scanner from you and I'm very impressed. 
I spent also some time for evaluating JTEC flashing, especially for Jeeps. 
Maybe as an Idea for your project: 
As "only" deactivating the SKIM / VATS function via EEEPROM, you can also deaktivate it via flashing. I can tell you which bits need to be changed to complete deactivate SKIM and VATS. 
Also Cruise Control changes (min. Start-Speed) VMAX changes, and many many other changes like fan control, spark, DTC off are possible. 

Lets have a talk if you are interested in sharing a few ideas :) 

KR 
Jannik

**Comment by laszlodaniel** [comment]
Hi Jannik,

thank you for your purchase and kind feedback!
JTEC configuration is sought by others here as well, so we welcome what you can share.

The EEPROM setting change is considered a clever workaround for security modules, as it puts the controller in an infinite learning mode. But the module has to be disconnected. The real deal would be turning security off in flash so that the module may remain connected.

I have JTEC/JTEC+ partial disassemblies ready to verify configuration bits.
Where do we look for the settings area?
